doc/endpoints/track.md

```diff
@@ -149,9 +164,8 @@ A promise that resolves with an array of objects, each containing audio feature
 
 ```typescript
 const trackIds = ['7ouMYWpwJ422jRcDASZB7P','4VqPOruhp5EdPBeR92t6lQ','2takcwOaAZWiXQijPHIx7B'];
-
-const audioFeaturesArray = await spotified.track.getMultipleTracksAudioFeatures(trackIds)
-console.log(audioFeaturesArray)
+const audioFeaturesArray = await spotified.track.getMultipleTracksAudioFeatures(trackIds);
```

Comment:
For this endpoint, we need to replace the parameters text with the following

- `ids` (required): An array of strings representing the unique Spotify IDs of the tracks for which audio analysis information is to be retrieved.